ckan/tests/controllers/test_user.py

```diff
@@ -123,6 +123,19 @@ def test_registered_user_login_bad_password(self):
                                    .format(user['fullname']))
 
 
+class TestLogout(helpers.FunctionalTestBase):
+
+    def test_user_logout(self):
```

Comment:
You're not testing the actually logging out (cos you are not logged in) just the redirect associated, so can you set the test title to reflect that? Otherwise this is great.